vue-app/src/store/index.ts

```diff
@@ -527,6 +524,14 @@ const getters = {
       (getters.isRoundContributionPhase || getters.isRoundReallocationPhase)
     )
   },
+  hasLowFunds(state: RootState): boolean {
```

Comment:
This should compare `currentUser.etherBalance` if `recipientRegistryInfo.depositToken` is ETH.

Reply:
I'll update.

Reply:
@corwintines I updated this but am realizing we may just want this as a getter in the RecipientSubmissionWidget component as it depends on `recipientRegistryInfo.depositToken`.

Other checks for `hasLowFunds` will depend on other criteria, e.g. on when contributing we should check for low DAI & ETH balance.

Reply:
Thats fair, I had that originally and then tried to abstract it out incase we were going to use it in other places. But I agree, there is probably a better case to have this done in the RecipientSubmissionWidget. I'll refactor that back in.

Reply:
Ive removed this getter and moved the low funds check into the RecipientSubmissionWidget. Can add the getter at a later time if there is a use for it.